.github/workflows/pr-staging-to-prod.yml

```
@@ -0,0 +1,61 @@
+name: 3 | Generate PR for releases/staging to releases/prod
+
+# This CI job is responsible for generating PRs that bring the HEAD of releases/staging into releases/prod.
+# These PRs are meant to be the only (standard) way that code is merged into the releases/prod branch.
+#
+# https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#keep-a-branch-up-to-date-with-another
+
+on:
+  workflow_dispatch:
+
+jobs:
+  prod-gen-pr:
+    name: 'Generate PR for merging to releases/prod branch'
+    runs-on: ubuntu-latest
+    environment:
+      name: release
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e
+        with:
+          token: ${{ secrets.RELEASE_SERVICE_ACCESS_TOKEN }}
+          ref: main
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e
+        with:
+          token: ${{ secrets.RELEASE_SERVICE_ACCESS_TOKEN }}
+          ref: releases/prod
+      - name: Reset promotion branch
+        run: |
+          git fetch origin releases/staging:releases/staging
+          git reset --hard releases/staging
+      - name: Setup git
+        run: |
+          git config user.name "UL Mobile Service Account"
+          git config user.email "hello-happy-puppy@users.noreply.github.com"
+      - uses: peter-evans/create-pull-request@ea54357f43e3d1cf1125471d0814f4d02cc0d364
+        id: cpr
+        with:
+          token: ${{ secrets.RELEASE_SERVICE_ACCESS_TOKEN }}
+          commit-message: "This PR merges latest commits of `releases/staging` to `releases/prod`. Upon approval and successful merge, a Github action associated with the prod Fastlane setup will deploy to TestFlight, where the app can be submitted to Apple."
```

```suggestion
          commit-message: "Merges latest commits of `releases/staging` into `releases/prod`."
```

You should update this to reflect web, not mobile.

---

Can this be left blank, considering that it is updated later?

---

Yeah I think so - deleting for now.